internal/controller/operatorconfigmap_controller.go

```diff
@@ -398,6 +399,16 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI() error {
 		return fmt.Errorf("unable to find the updated cluster version")
 	}
 
+	storageClients := v1alpha1.StorageClientList{}
+	if err := c.list(&storageClients); err != nil {
+		return fmt.Errorf("failed to list storage clients: %v", err)
+	}
```

Comment:
There is no reason the list the clients. The client CRs are already listed somewhere else in this controller

Reply:
Yes, But I can only use c.list() to list the storageclients, its not directly available in OperatorConfigMapReconciler
Do we want the storageclient list to be in OperatorConfigMapReconciler struct?